shipyard_blueprints/hubspot/shipyard_hubspot/hubspot_utils.py

```diff
@@ -222,16 +226,30 @@ def handle_import_file(
             headers = file.readline().strip().split(",")
 
             for header in headers:
-                if header == hubspot_alternate_id:
+                if (
+                    header == hubspot_alternate_id
+                ):  # Apply the unique id tag to email addresses
```

Comment:
Is there a reason to use `()` for the `if` check?

Reply:
That was added by our auto formatter black I believe for the line length. I will move the comment to its own line